sematic/types/types/huggingface/stored_model.py

```diff
@@ -71,6 +72,7 @@ def store(
             base_model_reference=base_model_reference,
         )
 
+    @lru_cache
```

Comment:
So that loading the model (which can take a long time) doesn't have to be done separately for both the interactive and non-interactive evaluations